src/main/java/bit/javaoop/Company.java

```diff
@@ -0,0 +1,46 @@
+package bit.javaoop;
+
+public class Company {
+    private String companyName;
+    private CEO ceo;
+
+
+    public void setCompanyName(String companyName) {
+        this.companyName = companyName;
+    }
+
+    public CEO hireCEO()
+    {
+        CEO ceo1 = CEO.getInstance();
+        ceo=ceo1;
+        ceo.setName("Krzychu"); //setting name e.g from keyboard
+        ceo.addEmployee(ceo);
+        return ceo;
+    }
+
+
+
+    public String toString() {
+        String hierarchy="";
+        for(AbstractEmployee e: ceo.getEmployeeList())
+        {
+            switch (e.getPosition())
+            {
```

Comment:
It would not work (the indentation would break) if the company had more advanced structure. For example if there was a Manager which has another Manager as it's employee and some non-Manager second employee.